app-admin/needrestart/needrestart-2.9.ebuild

```diff
@@ -0,0 +1,40 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+if [[ ${PV} == "9999" ]] ; then
+	EGIT_REPO_URI="git://github.com/liske/${PN}.git"
+	inherit git-r3
+	SRC_URI=""
+	KEYWORDS=""
+else
+	SRC_URI="https://github.com/liske/${PN}/archive/v${PV}.tar.gz"
```

Comment:
as an enhancement, we should use the [arrow notation](https://devmanual.gentoo.org/ebuild-writing/variables/index.html#renaming-sources) to rename the distfile as such:

```
SRC_URI="https://github.com/liske/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
```